internal/hcsoci/create.go

```diff
@@ -84,29 +191,79 @@ func CreateContainer(ctx context.Context, createOptions *CreateOptions) (_ cow.C
 	if coi.actualID == "" {
 		g, err := guid.NewV4()
 		if err != nil {
-			return nil, nil, err
+			return nil, err
 		}
 		coi.actualID = g.String()
 	}
 	if coi.actualOwner == "" {
 		coi.actualOwner = filepath.Base(os.Args[0])
 	}
 
-	if coi.Spec == nil {
```

Comment:
Do we not need this check anymore?

Reply:
This check was actually moved to `ValidateContainerConfig` so it wasn't needed here. However, I now realize that I have modified `initializeCreateOptions` to look at the `spec` and set late clone annotations so we might fail there if `spec` is nil. I will update the code to handle this.